src/directive/service/create.ts

```diff
@@ -20,6 +24,20 @@ export const createDirective = async (
     requestBody.emailFromAuthToken
   );
 
+  const accountResult = await db.sql<AccountResult>(
+    "directive.queries.check_account_by_email",
+    {
+      email: requestBody.stewardEmail,
+    }
+  );
```

Comment:
I don't think we should make this query if `type` isn't "transfer". Perhaps even better, we could avoid this query entirely by checking the insert query for an error on the `steward_account_id_not_null_when_transfer` constraint, the same way we handle invalid enum errors.

Reply:
That's a good point - thanks for pointing out the invalid enum errors. It should be ready for another look.